tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -132,19 +157,22 @@ func TestRandomizedSmall(t *testing.T) {
 
 	st := timetrack.StartTimer()
 
+	maxDirDepth := 3
+
 	opts := engine.ActionOpts{
 		engine.ActionControlActionKey: map[string]string{
 			string(engine.SnapshotDirActionKey):              strconv.Itoa(2),
 			string(engine.RestoreSnapshotActionKey):          strconv.Itoa(2),
-			string(engine.DeleteRandomSnapshotActionKey):     strconv.Itoa(1),
+			string(engine.DeleteRandomSnapshotActionKey):     strconv.Itoa(4),
 			string(engine.WriteRandomFilesActionKey):         strconv.Itoa(8),
-			string(engine.DeleteRandomSubdirectoryActionKey): strconv.Itoa(1),
+			string(engine.DeleteRandomSubdirectoryActionKey): strconv.Itoa(maxDirDepth),
 		},
 		engine.WriteRandomFilesActionKey: map[string]string{
-			fiofilewriter.IOLimitPerWriteAction:    strconv.Itoa(512 * 1024 * 1024),
-			fiofilewriter.MaxNumFilesPerWriteField: strconv.Itoa(100),
-			fiofilewriter.MaxFileSizeField:         strconv.Itoa(64 * 1024 * 1024),
-			fiofilewriter.MaxDirDepthField:         strconv.Itoa(3),
+			fiofilewriter.IOLimitPerWriteAction:        strconv.Itoa(512 * 1024 * 1024),
+			fiofilewriter.MaxNumFilesPerWriteField:     strconv.Itoa(100),
+			fiofilewriter.MaxFileSizeField:             strconv.Itoa(64 * 1024 * 1024),
+			fiofilewriter.MaxDirDepthField:             strconv.Itoa(maxDirDepth),
+			fiofilewriter.DeletePercentOfContentsField: strconv.Itoa(deleteContentsPercentage),
```

Comment:
What is the effect of this line (175) ?

Reply:
The delete-directory-contents action deletes some of the contents of a random directory, based on its input options values. `DeletePercentOfContentsField` is one of the input options. It sets the percentage for deleting contents of the random directory. So, setting this field and executing `DeleteDirectoryContentsActionKey` action would result in deletion of 50% of contents of a random directory in the source directory.

Reply:
And where is `DeletePercentOfContentsField` being used? which "action" uses this parameter?

Reply:
Good point. I added the `DeleteDirectoryContentsActionKey` that can use `DeletePercentOfContentsField`

Reply:
Where?

Reply:
Line 177

Reply:
Good. I had not seen that before.